lib/index.js

```diff
@@ -63,39 +66,133 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   app.get(
     basePath,
     optionallyAuthenticated,
-    //validate({query: 'services.account.getAccountsQuery'}),
-    asyncHandler(async (req, res, next) => {
-      if(req.query.exists === 'true' && req.query.email) {
+    // FIXME add an implementation in bedrock-express
+    boolParser(),
+    intParser(),
+    validate({query: 'account.get'}),
+    asyncHandler(async (req, res) => {
+      /**
+       * @func getAccounts
+       * @param {Request} req
+       * @param {Response} res
+       * @param {Object} req.query
+       * @param {String} req.query.email - required to acces this route
+       * @param {Boolean} req.query.exists - used by Registration
+       * @param {Number} req.query.limit
+       * @param  {String} req.query.cursor - an account id
+       * @description Gets account records (or checks for their existence).
+       * Visitors (not logged in) can confirm an email exists
+       * Admins (logged in) can list accounts by email
+       */
+      if(req.query.exists === true) {
         // allow anyone to check for account existence
         const {email} = req.query;
         const exists = await brAccount.exists({actor: null, email});
         if(exists) {
           return res.status(200).end();
         }
-        return res.status(404).end();
+        // TODO: improve error details
```

Comment:
What's our acceptance policy for `master` if there are TODO's?

Reply:
Judgment call. It's fine to have TODOs for future work in things that are otherwise working.

Reply:
TODO is also kind of vague. Not sure what more you want. the query is exists and if the account does not exist we return 404 not found with a message.

Reply:
@aljones15 we're not going to address this right now.  I added that because...  It is often the case that we want to provide helpful details in the error messages as appropriate.  So, perhaps including query parameters might be helpful when doing a NotFoundError etc.  Imagine yourself in the position of a dev reading a log file trying to determine why some API call is returning NotFoundError.

Reply:
it would be pretty simple to add the email to the Account ${email} does not exist. perhaps she gave you a fake number too?